vitals/Main.java

```diff
@@ -1,23 +1,82 @@
 package vitals;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public class Main {
-    static boolean batteryIsOk(float temperature, float soc, float chargeRate) {
-        if(temperature < 0 || temperature > 45) {
-            System.out.println("Temperature is out of range!");
-            return false;
-        } else if(soc < 20 || soc > 80) {
-            System.out.println("State of Charge is out of range!");
-            return false;
-        } else if(chargeRate > 0.8) {
-            System.out.println("Charge Rate is out of range!");
-            return false;
-        }
-        return true;
+
+  /**
+   *
+   */
+  private static final String CHARGE_RATE = "Charge Rate";
+  /**
+   *
+   */
+  private static final String STATE_OF_CHARGE = "State of Charge";
+  /**
+   *
+   */
+  private static final String TEMPERATURE = "Temperature";
+
+  static Map<String, Boolean> checkVitalsStatus(final Map<String, Float> vitalValuesMap) {
```

Comment:
Naming of methods can be more intuitive. Here the method is returning a map and not just checking.
Also the method BatteryIsOk() is a single API that provides the status which is a function of all parameters.
By not having such method, the user is expected to know each parameter and verify each parameter separately. Please see if not having that method is convenient or not.